src/nekontrol/interactive/cli.py

```diff
@@ -1,12 +1,46 @@
-import os.path as path
-
 import click
 
+from nekontrol.config import Config, exec_config
 from nekontrol.console import setup_console
-from nekontrol.config import exec_config
+
 from . import commands
 
-executable_file = click.Path(exists=True, readable=True, file_okay=True, dir_okay=False)
+executable_file = click.Path(
+    exists=True, readable=True, file_okay=True, dir_okay=False, resolve_path=True
+)
+
+
+def config_parser(path_argument: str):
+    def inner(f):
+        @click.option("--diff/--no-diff", default=None, help="Display diff")
+        @click.option("--verbose/--no-verbose", default=None, help="Verbose output")
+        @click.option(
+            "--ignore-debug/--no-ignore-debug",
+            default=None,
+            help="Ignore lines starting with `debug`",
+        )
+        @click.option(
+            "--force/--no-force",
+            default=None,
+            help="Submit even if sample verification fails",
+        )
```

Comment:
this means that all commands have a `--force`, yes? not sure that's what we want. especially when the help text talks about submitting.

Reply:
sorry i missed your comment before reviewing :x